taxi/forms.py

```diff
@@ -50,3 +51,36 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
```

Comment:
The comment about regex validation is not actionable here. Consider implementing regex validation if needed or remove the comment.